src/main/java/com/internet/shop/exceptions/DataException.java

```diff
@@ -0,0 +1,7 @@
+package com.internet.shop.exceptions;
+
+public class DataException extends RuntimeException {
```

Comment:
` Let’s create custom exception DataProcessingException with two parameters: String message and Throwable ex`. Don't know will it be a mistake with the exception name, but it what was said in the checklist